shell/platform/fuchsia/flutter/flatland_platform_view.cc

```diff
@@ -153,7 +153,7 @@ void FlatlandPlatformView::OnChildViewViewRef(
   fuchsia::ui::views::ViewRef view_ref_clone;
   fidl::Clone(view_ref, &view_ref_clone);
   pointer_injector_delegate_->OnCreateView(view_id, std::move(view_ref_clone));
-
+  OnChildViewConnected(content_id);
   child_view_info_.at(content_id)
```

Comment:
You can add the TODO here to remove the subsequent calls.

Reply:
Done